dts/bindings/bluetooth/infineon,cyw43xxx-bt-hci.yaml

```diff
@@ -18,6 +18,9 @@ description: |
                      &p3_2_scb2_uart_rts &p3_3_scb2_uart_cts>;
         pinctrl-names = "default";
 
+        /* HW Flow control must be enabled for HCI H4 */
+        hw-flow-control;
```

Comment:
perhaps add a BUILD_ASSERT to the hci driver to check that this condition has been met?

Reply:
updated